src/main.js

```diff
@@ -1,20 +1,40 @@
 import App from './App.vue'
-import '@/global/components'
-import '@/global/filters'
-import { configuredRouter as router, configuredVuetify } from './plugins'
+import registerComponents from '@/global/components'
+import { api, i18n, returnTo, router, user, vuetify } from './plugins'
+import { createApp } from 'vue'
 import * as Sentry from '@sentry/vue'
-import Vue from 'vue'
 import VueSanitize from 'vue-sanitize-directive'
 
-Vue.use(VueSanitize)
-
 const dsn = import.meta.env.VITE_SENTRY_DSN
 const release = __APP_VERSION__
 
+const app = createApp(App)
+
+app.config.globalProperties.$idpConfig = {}
+;(async () => {
```

Comment:
looks strange to me with a semicolon at the front

Reply:
It has to be there because javascript is bad, but if you prefer this would be equivalent
`async function getConfig() {
  try {
    const config = await api.get('config')
    app.config.globalProperties.$idpConfig = config
  } catch (error) {
    console.error('Failed to load IDP configuration:', error)
  }
}
await getConfig()`

Reply:
I would prefer that, but don't place any weight on my opinion. I'm not really a javascript programmer.